[Tracer][.NET] Small improvements to setup doc (#15713)
DataDog/documentation: content/en/tracing/trace_collection/dd_libraries/dotnet-core.md

2. Run the .NET Tracer MSI installer with administrator privileges.

> <div class="alert alert-info">
>   <div class="alert-info">
>     <div>You can also script the MSI setup. A powershell example would be: <br/>
>     <code>RUN Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-apm.msi'</code></div>
>   </div>
> </div>

Reviewer: ```suggestion
You can also script the MSI setup by running the following in PowerShell: `RUN Start-Process -Wait msiexec -ArgumentList '/qn /i datadog-apm.msi'`
```
